cr-examples/hub-of-hubs.open-cluster-management.io_hoh_heartbeat_cr.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: hub-of-hubs.open-cluster-management.io/v1
+kind: HohHeartbeat
+metadata:
+  name: hub-of-hubs-heartbeat
+  labels:
+    name: hub-of-hubs-heartbeat
+status:
```

Comment:
the status should not be in the examples, kubectl cannot update the status anyway. I mean just remove the status field.

Reply:
done